### PR TITLE
Home16

### PR DESCRIPTION
**Pull request by IgorPrihodko**

### CODE REVIEW

File: src/main/java/homework16/MailService.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package homework16;
+
+import java.util.*;
```
Comment (Sofasmile) [inline src/main/java/homework16/MailService.java]: try doesn't use auto import

File: src/main/java/homework16/MailService.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.function.Consumer;
+
+public class MailService<T> implements Consumer<Sendable<T>> {
+    @Override
+    public void accept(Sendable<T> tSendable) {
+        List<T> list;
+        list = mailBox.get(tSendable.getTo());
```
Comment (Sofasmile) [inline src/main/java/homework16/MailService.java]:
```suggestion
        List<T> list = mailBox.get(tSendable.getTo());
```

File: src/main/java/homework16/MailService.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        list.add(tSendable.getContent());
+        mailBox.put(tSendable.getTo(), list);
+    }
+
+    private Map<String, List<T>> mailBox = new HashMap<String, List<T>>() {
+        @Override
+        public List<T> get(Object key) {
+            if (!this.containsKey(key)) {
```
Comment (Sofasmile) [inline src/main/java/homework16/MailService.java]: we can use method getOrDefault() to simplify code

File: src/main/java/homework16/MailService.java (hunk 1 above)
Comment (IgorPrihodko) [reply src/main/java/homework16/MailService.java]: fixed

File: src/main/java/homework16/MailService.java (hunk 2 above)
Comment (IgorPrihodko) [reply src/main/java/homework16/MailService.java]: fixed

File: src/main/java/homework16/MailService.java (hunk 3 above)
Comment (IgorPrihodko) [reply src/main/java/homework16/MailService.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**